### DISCUSSION TITLE
what's "a popular open source project"

### DISCUSSION THREAD

**Discussion by trim21** [community/community#30052] [Copilot Conversations]
I happened to be a maintainer (with write access) of a open source project with 20k stars, and I found that I didn't get a copilot subscription.

I'd like to ask that's the standard of "popular open source project" or there is a cron to give permission?

**Comment by glcraft** [comment]
I don't have a lot of stars but I'm curious about what is required to have a copilot open-source subscription.

**Comment by duarch** [comment]
> a open source project with 20k stars

@Trim21 Which one?

**Reply by trim21** [reply]
gofiber/fiber

**Comment by toddtarsi** [comment]
Yeah, I feel similar to this. I basically wrote v4 of the Selenium IDE from scratch over the last seven months, but I guess that isn't good enough. Funny enough, its not even about the ten stupid bucks, but also it's like "my last seven months aren't worth ten stupid bucks". The other side of it that pisses me off is that I've put thousands of lines of open source code out there which is good enough for you to harvest for your bullshit AI, but not good enough to benefit from my own hard work 😠

**Comment by MulleDK19** [comment]
They say you get it for free, but the requirements are so narrow and vague, it's only a small handful of people.

You don't just need to be the maintainer of an extremely popular repository, you need to be the admin.

**Reply by trim21** [reply]
Not sure about the second paragraph…

I mentioned write permission because of this

> A maintainer of a popular open source project is defined as someone who has **write or** admin access to one or more of the most popular open source projects on GitHub. 

Quote from free use FAQ

https://github.com/pricing#i-work-on-open-source-projects-can-i-get-access-to-github-copilot-for-free
